conf/machine/include/rpi-base.inc

```diff
@@ -77,6 +77,7 @@ MACHINE_FEATURES += "apm usbhost keyboard vfat ext2 screen touchscreen alsa blue
 MACHINE_FEATURES_BACKFILL_CONSIDERED = "rtc"
 
 MACHINE_EXTRA_RRECOMMENDS += " kernel-modules udev-rules-rpi"
+MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " ${@oe.utils.conditional("ENABLE_I2C", "1", "kernel-module-i2c-dev kernel-module-i2c-bcm2708", "", d)}"
```

Comment:
why dont we add it to `MACHINE_EXTRA_RRECOMMENDS` ?

Reply:
According to Yocto mega manual variable `MACHINE_EXTRA_RRECOMMENDS` affects only images based on `packagegroup-base`, which does **not** include the `core-image-minimal` or `core-image-full-cmdline` images. [For more details please have a look here](https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#var-MACHINE_EXTRA_RRECOMMENDS).

Thanks, Leon

Reply:
then lets convert the `MACHINE_EXTRA_RRECOMMENDS` to `MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS` while you are here.

Reply:
OK, sounds good to me. I will update the GitHub pull request. Just give me a moment. Thanks for the feedback!